src/ansys/mechanical/core/run.py

```diff
@@ -142,6 +146,13 @@ def _cli_impl(
         args.append("-script")
         args.append(input_script)
 
+    if script_args:
+        args.append("-ScriptArgs")
+        if '"' in script_args:
```

Comment:
I don't fully understand the purpose of this if expression

Reply:
@klmcadams yes it has to be wrapped in quotes. Is that what this is doing?

Reply:
@koubaa It checks if there's a double quote in the script_args, and if so, it'll wrap it in single quotes. Otherwise, it'll wrap it in double quotes

If --script-args='"arg1",arg2', then it should wrap the arg in single quotes since the script_args contain double quotes. But for --script_args="arg1,arg2", it'll wrap it in double quotes

Reply:
@klmcadams can you add unit tests to cover these cases?